### PR TITLE
ItemDB Updates:

### PR DESCRIPTION
**Pull request by Jedzkie**
- Change ID to Constants of some items both pre-re and re.
 - Corrected some item names and scripts.
 - Added 'Mysterious_Can2', 'Mysterious_PET_Bottle2', '2009_Rice_Cake_Soup', 'Vulcan Bullet Magazine', 'Unripe_Yggdrasil', 'Iggdrasilberry_Box' effect.

### CODE REVIEW

File: db/re/item_db.conf [hunk 1]
```diff
@@ -84724,7 +84744,7 @@ item_db: (
 {
 	Id: 12642
 	AegisName: "Fruit_Of_Mastela_Box2"
-	Name: "Fruit Box (100)"
+	Name: "Mastela Fruit 100 Box"
```
Comment (dastgir) [inline db/re/item_db.conf]: Mastela Fruit Box 100 maybe?

File: db/pre-re/item_db.conf [hunk 2]
```diff
@@ -66942,7 +66952,7 @@ item_db: (
 	Job: 0x00000400
 	Upper: 8
 	BuyingStore: true
-	Script: <" //madoheal rand(300,400),0; ">
+	Script: <" if(checkmount() == MOUNT_MADO) itemheal rand(300,400), 0; ">
```
Comment (dastgir) [inline db/pre-re/item_db.conf]: Mado in pre-renewal?
Seems doubtful

Comment (Jedzkie) [inline db/pre-re/item_db.conf]: 3rd jobs are enabled in pre-re, what if server admin uses 3rd job in pre-renewal? Mechanics cannot use that item.

Comment (AnnieRuru) [inline db/pre-re/item_db.conf]: I remember I read a blog in rathena saying ... 3rd job never exist in pre-renewal
https://rathena.org/board/blog/23/entry-87-pre-renewal-rathena/
if we want to follow it officially, yeah pre-renewal shouldn't even have any 3rd job stuff
I don't play official server anymore so don't ask  me about official things

Comment (Jedzkie) [inline db/pre-re/item_db.conf]: yes, in official, 3rd job never exist in pre-renewal. but in our System, it is enabled but it has limitations.

Comment (dastgir) [inline db/pre-re/item_db.conf]: @MishimaHaruna

**Comment by MishimaHaruna** [comment]
Merging, thank you!

**Pull request closed without merging**